ci-operator/step-registry/baremetal/lab/post/agent/baremetal-lab-post-agent-commands.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+set -o nounset
+set -o errexit
+set -o pipefail
+
+SSHOPTS=(-o 'ConnectTimeout=5'
+  -o 'StrictHostKeyChecking=no'
+  -o 'UserKnownHostsFile=/dev/null'
+  -o 'ServerAliveInterval=90'
+  -o LogLevel=INFO
+  -i "${CLUSTER_PROFILE_DIR}/ssh-key")
+
+ssh "${SSHOPTS[@]}" root@openshift-qe-055.arm.eng.rdu2.redhat.com <<EOF
+cd /root/workdir/agent-bm-deployments/
+ansible-playbook -i prow_inventory dns-cleanup.yaml
```

Comment:
Seem like `dns-cleanup.yaml` and `install.yaml` only available in `openshift-qe-055.arm.eng.rdu2.redhat.com`, right?

Reply:
Yes, I had thought about using an image or multiple steps to handle that so the installation program is also under code review, but that makes things difficult because the installation manifests needs to take hostnames, IP addresses, credentials, ssh-key, pull secrets. There are some security implications to do it that way. 

A good solution is WIP: QE-BM-Infra-RDU2, but this solution probably won't complete during our 4.12 timeframe